packages/action-menu/stories/index.ts

```diff
@@ -52,6 +54,9 @@ export const ActionMenuMarkup = ({
             <sp-menu-divider></sp-menu-divider>
             <sp-menu-item>Save Selection</sp-menu-item>
             <sp-menu-item disabled>Make Work Path</sp-menu-item>
+            ${tooltip_description
+                ? html`<sp-tooltip slot='tooltip' self-managed placement=${tooltip_placement}>${tooltip_description}</sp-tooltip`
```

Comment:
There's no definition for `<sp-tooltip>` included in this file.

There's also a missing `>`.